### PR TITLE
Add Kb for "rwx mount ownership set to nobody" issue

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/2357

### CODE REVIEW

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+author: Chin-Ya Huang
+draft: false
+date: 2021-03-31
+categories:
+  - "rwx"
+---
+
+## Applicable versions
+
+All Longhorn versions.
```
Comment (innobead) [inline content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md:12]: Actually just 1.1.0 which starts having RWX support. Please double confirm.

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## Symptoms
+
+When Pod mounts with RWX volume, the Pod share mount directory and all of the ownership of its recurring contents are shown as nobody, and in the share-manager is shown as root.
```
Comment (innobead) [inline content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md:16]: , but in the share manager is shown as root.

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+drwx------ 2 root root 16384 Mar 31 04:42 lost+found
+```
+
+## Background
+
+The nfs-ganesha in share-manager uses idmapd for NFSv4 ID mapping and is set to use `localdomain` as its export Domain.
+
+## Reason
+
+A result of content mismatch in /etc/idmapd.conf between client(share-manager) and server(host) cause ownership to change.
```
Comment (innobead) [inline content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md:34]: Client is host, server is share manager pod.

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM 

But some comments to resolve

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md (hunk 1 above)
Comment (c3y1huang) [reply content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md]: Yes you are right, https://github.com/longhorn/longhorn/releases/tag/v1.1.0.

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md (hunk 2 above)
Comment (c3y1huang) [reply content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md]: Fixed

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md (hunk 3 above)
Comment (c3y1huang) [reply content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md]: Thanks, fixed.

**Review by innobead — DISMISSED** [review]
LGTM

@catherineluse please help review. Thanks!

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+drwx------ 2 root root 16384 Mar 31 04:42 lost+found
+```
+
+## Background
+
+The nfs-ganesha in share-manager uses idmapd for NFSv4 ID mapping and is set to use `localdomain` as its export Domain.
+
+## Reason
+
+A result of content mismatch in /etc/idmapd.conf between client(host) and server(share-manager) cause ownership to change.
```
Comment (catherineluse) [inline content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md:34]: causes ownership

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Pipefs-Directory = /run/rpc_pipefs
+# set your own domain here, if it differs from FQDN minus hostname
+Domain = localdomain
+
+[Mapping]
+
+Nobody-User = nobody
+Nobody-Group = nogroup
+```
+2. Delete, and re-create RWX resource stack (pvc + pod).
```
Comment (catherineluse) [inline content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md:75]: Delete and recreate

**Review by catherineluse — CHANGES_REQUESTED** [review]
Looks good - left a couple of comments

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md (hunk 4 above)
Comment (c3y1huang) [reply content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md]: Thanks, fixed.

File: content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md (hunk 5 above)
Comment (c3y1huang) [reply content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md]: Thanks, fixed.

**Comment by c3y1huang** [comment]
I've updated the Applicable versions since we are applying [fix](https://github.com/longhorn/longhorn-share-manager/pull/11) in v1.1.1.

**Review by innobead — APPROVED** [review]

**Review by catherineluse — APPROVED** [review]

**Pull request merged**